.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,72 @@
+name: Tests
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-16.04
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        php: [5.4, 8.0, 7.4, 7.3, 7.2, 7.1, 7.0, 5.6, 5.5]
```

Comment:
minor `nit` on ordering.

```suggeston
        php: [5.4, 5.5, 5.6, 7.0, 7.1, 7.2, 7.3, 7.4, 8.0]
```

Reply:
all the 5.6 and most of 7.x are dead now also right?

Reply:
Danack asked for a 5.4 step before the matrix, as this is more likely to fail and he is conscious of wasteful energy usage. I'm not aware of a clean solution for this which can avoid duplication, so for now I moved it first in the row.

Reply:
5.4 is the first in the list in both cases, so I don't think there is much of a difference, also because the first few jobs from the matrix are started in parallel anyway.

Reply:
The ordering is about readability, specifically scanability, being able to scan in a direction. It just lowers mental load...